app/controllers/work_days_controller.rb

```diff
@@ -0,0 +1,96 @@
+class WorkDaysController < ApplicationController
+  before_action :set_work_day, only: [:show, :edit, :update, :destroy]
+
+  def index
+    if params.has_key?(:month) && params.has_key?(:year)
+      @month = params[:month].to_i
+      @year = params[:year].to_i
+      @project = params.has_key?(:project) ? Project.find(params[:project].to_i) : nil
+      @time_sheet = time_sheet_for(@year, @month, @project)
+      @work_days = all_for(@year, @month, @project)
+    else
+      date = Date.today
+      redirect_to work_days_path(month: date.month, year: date.year)
+    end
+  end
+
+  def show
+  end
+
+  def new
+    @work_day = WorkDay.new
+  end
+
+  def edit
+  end
+
+  def create
+    @work_day = WorkDay.new(work_day_params)
+    @work_day.user_id = current_user.id
+
+    if @work_day.save
```

Comment:
Kannst du, wie in den anderen Controllern auch, noch Flash-Messages hinzufügen? :)